rali/rali/source/rali_api_augmentation.cpp

```diff
@@ -1653,7 +1685,10 @@ raliResizeCropMirrorFixed(
         output_info.height(dest_height);
         output = context->master_graph->create_image(output_info, is_output);
         output->reset_image_roi();
-        context->master_graph->add_node<ResizeCropMirrorNode>({input}, {output})->init(crop_h, crop_w, mirror);
+        std::shared_ptr<ResizeCropMirrorNode> rcm_node =  context->master_graph->add_node<ResizeCropMirrorNode>({input}, {output});
```

Comment:
Codacy found an issue: [Either the condition '!context' is redundant or there is possible null pointer dereference: context.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5976200)